shopinvader_sale_packaging/tests/test_cart.py

```diff
@@ -4,8 +4,10 @@
 
 from odoo.addons.shopinvader.tests.common import CommonCase
 from odoo.addons.shopinvader.tests.test_cart_item import ItemCaseMixin
+from odoo.tests import tagged
 
 
+@tagged("post_install", "-at_install")
```

Comment:
@simahawk Can we avoid running tests @post_install?

Reply:
let's see w/ https://github.com/shopinvader/odoo-shopinvader/pull/783/commits/b19f9f59d02cd1c7ec2a5a71d8833a3240116c4e